Tests/AppTests/Logger/LogzIOLoggerTests.swift

```diff
@@ -0,0 +1,54 @@
+import XCTest
+@testable import App
+
+class LogsTests: XCTestCase {
+    func test_logzio_json_formatting() throws {
+        let message = LogzIOLogger.LogMessage(
+            timestamp: LogsTests.fixedDate,
+            message: LogsTests.message,
+            level: .debug,
+            file: "somefile.swift",
+            function: "somefunction",
+            line: 1337,
+            column: 42
+        )
+
+        let serializer = JSONEncoder()
+        let data = try serializer.encode(message)
+
+        print(message)
```

Comment:
Do we need this print?

Reply:
Yeah, that's the whole goal of the Logger 😛 .

Any custom Logger can do whatever they want, like sending the logs to a server or print it to the console or whatnot. Here because the way our services are integrated with Logz.io in our infra is that Logz.io picks things that are logged to STDOUT, we need to put that string to stdout, hence the `print` 😉 without it nothing would get logged to Logz.io.

I'll add a comment

Reply:
Fixed in 8953830

Reply:
🤦‍♂ I just saw that this comment was in the Tests code, not in the main implementation code… now your comment makes way more sense… removing this useless `print` now 😉